composer.json

```diff
@@ -15,28 +15,38 @@
             "homepage": "https://github.com/sonata-project/SonataCoreBundle/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/Ener-Getick/PHP-CS-Fixer"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/Ener-Getick/exporter"
+        }
```

Comment:
Must be removed once https://github.com/sonata-project/exporter/pull/55 is merged.